crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -86,41 +98,62 @@ impl Mesh {
 
     /// Sets the data for a vertex attribute (position, normal etc.). The name will
     /// often be one of the associated constants such as [`Mesh::ATTRIBUTE_POSITION`].
-    pub fn set_attribute(
+    #[inline]
```

Comment:
Did you test with/without inline and compare?

Reply:
I didn't, but as a general rule of thumb I inline single line "passthrough" calls for collection types and field accessors. Pretty much every time I have benchmarked that pattern, I see some improvement (and it is what `std` does generally).